### ISSUE TITLE
Style/MixinUsage false positive in specs

### ISSUE THREAD

**Issue by tagliala** [bbatsov/rubocop#5059]
The Style/MixinUsage finds false positive inside specs on the current master branch of RuboCop

--------

## Expected behavior

Does not report a lint

## Actual behavior

Reports a lint

#### my_class_spec.rb
```rb
# frozen_string_literal: true

require 'spec_helper'

# Top level docs
class MyClass
  def self.descendants
    [String]
  end
end

describe MyClass do
  it 'includes class' do
    described_class.descendants.should include String
  end
end
```

```
$ rubocop my_class_spec.rb 
Inspecting 1 file
C

Offenses:

my_class_spec.rb:14:40: C: include is used at the top level. Use inside class or module.
    described_class.descendants.should include String
                                       ^^^^^^^^^^^^^^

1 file inspected, 1 offense detected
```

Real world use case: https://github.com/brigade/scss-lint/blob/master/spec/scss_lint/reporter_spec.rb

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.51.0 (using Parser 2.4.0.2, running on ruby 2.4.2 x86_64-darwin16)
```

**Comment by koic** [comment]
Thanks for opening issue. I can reproduce it.

I think that this minimum case will be below.

```ruby
do_something(include(M))
```

The difficulty is that it can not be judged by receiver object whether it is `Module#include` or other` include` methods.

However, as indicated in the issue example, there are some cases where it becomes false positive with RSpec.
On the other hand, I think that there are not many use cases to use `Module#include` as method arguments.

So it may be a practical solution to make `do_something(include(M))` case false negative.

**Comment by Drenmi** [comment]
I agree @koic. AFAIK, if the `include` call is a method argument, it's safe to assume it's not a mixin.

This would also mean the fix should be trivial:

```ruby
return if include_node.argument?
```

**Comment by koic** [comment]
@Drenmi Thanks for comment and advice. I will open a PR to resolve this issue.

**Comment by tagliala** [comment]
Thanks!
